<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>组播端口</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboIP">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>78</y>
      <width>481</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinPort">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>9</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>35320</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>9</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>组播地址</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editMsg">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>44</y>
      <width>311</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>123abc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>418</x>
      <y>41</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>组播消息</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actStart"/>
   <addaction name="actStop"/>
   <addaction name="separator"/>
   <addaction name="actHostInfo"/>
   <addaction name="separator"/>
   <addaction name="actClear"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actStart">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/620.bmp</normaloff>:/images/images/620.bmp</iconset>
   </property>
   <property name="text">
    <string>加入组播</string>
   </property>
  </action>
  <action name="actStop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/624.bmp</normaloff>:/images/images/624.bmp</iconset>
   </property>
   <property name="text">
    <string>退出组播</string>
   </property>
  </action>
  <action name="actClear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/212.bmp</normaloff>:/images/images/212.bmp</iconset>
   </property>
   <property name="text">
    <string>清空文本</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actHostInfo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/110.JPG</normaloff>:/images/images/110.JPG</iconset>
   </property>
   <property name="text">
    <string>actHostInfo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actClear</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
